### PR TITLE
Export and benchmark build_jumpdest_map()

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Review by yperbasis — APPROVED** [review]

File: lib/evmone/baseline.cpp [hunk 1]
```diff
@@ -23,11 +19,13 @@ JumpdestMap build_jumpdest_map(const uint8_t* code, size_t code_size)
         if (op == OP_JUMPDEST)
             m[i] = true;
         else if (op >= OP_PUSH1 && op <= OP_PUSH32)
-            i += static_cast<size_t>(op - OP_PUSH1 + 1);
+            i += op - size_t{OP_PUSH1 - 1};
```
Comment (axic) [inline lib/evmone/baseline.cpp:22]: Wonder why don't we just have some kind of `push_size()` helper if this is used in other places too?

Comment (chfast) [reply lib/evmone/baseline.cpp:22]: Is it?

Comment (axic) [reply lib/evmone/baseline.cpp:22]: Not sure, is it?

Comment (chfast) [reply lib/evmone/baseline.cpp:22]: I have not found other case in lib/evmone.

**Pull request merged**